oc_dms_mirror/rest_api/app/routes.py

```
@@ -0,0 +1,43 @@
+import json
+from flask import Response, request, current_app
+from . import dms_mirror_bp
+from oc_dms_mirror.dms_mirror import DmsMirror
+import logging
+
+
+def get_dms_mirror():
+    _dmsMirror = DmsMirror()
+    _dmsMirror.init(current_app.args)
+    return _dmsMirror
+
+
+def response_json(code, data):
+    """
+    Return JSON data response
+    :param int code: HTTP response code
+    :param data: dict or list to send as response content
+    """
+    if not isinstance(data, str):
+        data = json.dumps(data)
+
+    return Response(
+        status=code,
+        mimetype='application/json',
+        response=data)
+
+
+@dms_mirror_bp.route('/register-component-version-artifact', methods=['POST'])
+def register_component_version_artifact():
+    """
+    Endpoint performing component/version sync with DMS on demand.
+    """
+    logging.info(f"POST {request.url_rule.rule} from [{request.remote_addr}]")
+    try:
+        _component = request.json.get('component')
+        _version = request.json.get('version')
```

1. Here we want to raise an exception in case any parameter  omitted. Please use square brackets.
2. Please do not assign a value to variable if You use it once in the method below. Use direct value instead:
`get_dms_mirror().process_version(request.json['component'], request.json['version'])`